supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/authentication/JesCredentials.scala

```diff
@@ -0,0 +1,5 @@
+package cromwell.backend.impl.jes.authentication
+
+import com.google.api.client.auth.oauth2.Credential
+
+case class JesCredentials(genomicsCredential: Credential, gcsCredential: Credential)
```

Comment:
I called it gcs because it's the one coming from `filesystems.gcs.auth` in the config but if `userAuthCredential` makes it clearer I'm all for that !